machine-runner/src/design/protocol.ts

```diff
@@ -37,25 +37,37 @@ export type SwarmProtocol<
 export namespace SwarmProtocol {
   /**
    * Construct a SwarmProtocol
-   * @param swarmProtocolName - The name of the swarm protocol
+   * @param swarmName - The name of the swarm protocol
    * @param tags - the tags used to mark the events passed to Actyx
    * @param registeredEventFactories - MachineEvent.Factories that are allowed
    * to be used for communications in the scope of this SwarmProtocol
    * @example
-   * const protocol = SwarmProtocol.make("")
+   * const HangarDoorTransitioning = MachineEvent
+   *   .design("HangarDoorTransitioning")
+   *   .withPayload<{ fractionOpen: number }>()
+   * const HangarDoorClosed = MachineEvent
+   *   .design("HangarDoorClosed")
+   *   .withoutPayload()
+   * const HangarDoorOpen = MachineEvent
+   *   .design("HangarDoorOpen")
+   *   .withoutPayload()
+   *
+   * // Creates a protocol
+   * const HangarBay = SwarmProtocol.make('HangarBay', ['hangar-bay'], Events.all)
```

Comment:
`Events.all` isn’t declared above, and `as const` doesn’t yet work, so I think this needs to list the events for now

Reply:
Ah that's right. Thanks for catching this

Reply:
Fixed in https://github.com/Actyx/machines/pull/29/commits/85ed0b3f30efb6990caf97ab1e6de1b826f78653